library/lib/build.js

```diff
@@ -231,6 +241,13 @@ module.exports = function build({ watch }) {
         })
       ],
     }
+
+    function externalConfForModulesDir (modulesDir = 'node_modules') {
+      return {
+        whitelist: ['@kaliber/config', kaliberBuildClientModules, /\.css$/],
+        modulesDir
```

Comment:
We will have to check the effect of setting `modulesDir`. Normally this library (`webpack-node-externals`) uses the `package.json` from the `cmd()`. I'll have to check the source code for the effect.

Reply:
I was wrong, the library reads it from disk by default instead of `package.json`